test/e2e/cluster.go

```diff
@@ -114,6 +116,13 @@ var _ = Describe("Cluster", func() {
 })
 
 func createStatefulSet(ctx context.Context, cli kubernetes.Interface) error {
+	oc, _ := clients.OpenshiftClusters.Get(ctx, vnetResourceGroup, clusterName)
+	installVersion, _ := version.ParseVersion(*oc.ClusterProfile.Version)
+
+	defaultStorageClass := "managed-premium"
+	if installVersion.V[0] == 4 && installVersion.V[1] == 11 {
```

Comment:
I wasn't sure, but should I open up this conditional to match >= 4.11?  Not that it matters at this time, but when 4.12 comes to ARO we would have to update this.  I just haven't had the chance to check what the default storage classes are for 4.12 to confirm.